src/delivery-options/config/configBus.js

```diff
@@ -390,15 +390,16 @@ export const createConfigBus = (eventCallee = null) => {
        * @param {MyParcel.CarrierName|Number} obj.value - New carrier or pickup location id.
        */
       updateCurrentCarrier({ name, value }) {
-        switch (name) {
-          case FORM.CARRIER:
-            this.currentCarrier = value;
-            break;
-          case FORM.PICKUP_LOCATION:
-            if (this.pickupLocations.hasOwnProperty(value)) {
-              this.currentCarrier = this.pickupLocations[value].carrier;
-            }
-            break;
+        if (FORM.CARRIER === name) {
```

Comment:
Aangezien we destructeren, lijkt het mij ook verstanding om te checken of de name of value wel bestaan

Reply:
Die bestaan altijd :)